### PR TITLE
Disable explicit ssl for management

### PR DESCRIPTION
**Pull request by ironpinguin**
Ticket MODULES-1974

### CODE REVIEW

**Comment by bmjen** [comment]
@ironpinguin Thanks for the contribution! Can you please rebase to pull in some fixes to make tests pass? Thanks.

**Comment by ironpinguin** [comment]
Rebased. Now all test are running again :-)

File: lib/puppet/provider/rabbitmq_binding/rabbitmqadmin.rb [hunk 1]
```diff
@@ -39,7 +39,7 @@ def self.instances
     resources = []
     all_vhosts.each do |vhost|
       all_bindings(vhost).collect do |line|
-        source_name, destination_name, destination_type, routing_key, arguments = line.split(/\t/)
+        source_name, destination_name, destination_type, routing_key, arguments = line.split()
```
Comment (bmjen) [inline lib/puppet/provider/rabbitmq_binding/rabbitmqadmin.rb]: Is this a necessary change, does not seem pertinent to adding the new `management_ssl` param.

**Comment by bmjen** [comment]
Hi @ironpinguin, I left more feedback, also.. can you add at least unit tests for the new param? Thanks!

**Comment by ironpinguin** [comment]
I will remove the additional change and add also unit tests for the new param asap. :-)

**Comment by bmjen** [comment]
Thanks @ironpinguin!

**Comment by ironpinguin** [comment]
Sorry for the delay but now i add the missing unit tests. it this good enough for you?

**Comment by jonnytpuppet** [comment]
Closing due to #368

**Pull request closed without merging**